lang/ruby/build.sh

```diff
@@ -24,11 +24,6 @@ cd "$(dirname "$0")"
 export GEM_HOME="$PWD/.gem/"
 export PATH="/usr/local/rbenv/shims:$GEM_HOME/bin:$PATH"
 
-# bootstrap bundler
-gem install --no-document -v 1.17.3 bundler
-
-# rbenv is used by the Dockerfile but not the Github action in CI
-rbenv rehash 2>/dev/null || echo "Not using rbenv"
```

Comment:
Use of rbenv was previously removed from the Dockerfile setup: https://github.com/apache/avro/commit/cf7bfe7710febdc84e70fcc8ad21d859dbf1be1f#diff-9ad53a2cdd23436f405be61d20306403700f362287b3d86609319919176a6fb6L191-L203